applications/HDF5Application/python_scripts/xdmf_utils.py

```diff
@@ -0,0 +1,311 @@
+"""Utilities for creating XDMF metadata from results stored in HDF5.
+
+XDMF utilities are separate from the core xdmf to reduce core dependencies.
+
+See:
+- core.xdmf
+
+license: HDF5Application/license.txt
+"""
+
+
+import xml.etree.ElementTree as ET
+import os
+import warnings
+from itertools import chain
+from contextlib import contextmanager
+import re
+
+
+import KratosMultiphysics
+import KratosMultiphysics.HDF5Application as KratosHDF5
+from KratosMultiphysics.HDF5Application.core.xdmf import SpatialGrid
+from KratosMultiphysics.HDF5Application.core.xdmf import HDF5UniformDataItem
+from KratosMultiphysics.HDF5Application.core.xdmf import Geometry
+from KratosMultiphysics.HDF5Application.core.xdmf import TopologyCellType
+from KratosMultiphysics.HDF5Application.core.xdmf import UniformMeshTopology
+from KratosMultiphysics.HDF5Application.core.xdmf import UniformGrid
+from KratosMultiphysics.HDF5Application.core.xdmf import NodalData
+from KratosMultiphysics.HDF5Application.core.xdmf import ElementData
+from KratosMultiphysics.HDF5Application.core.xdmf import TemporalGrid
+from KratosMultiphysics.HDF5Application.core.xdmf import Time
+from KratosMultiphysics.HDF5Application.core.xdmf import Domain
+from KratosMultiphysics.HDF5Application.core.xdmf import Xdmf
```

Comment:
wouldn't it be easier to import it once and then use it?
=>
~~~py
import KratosMultiphysics.HDF5Application.core.xdmf as hdf5_xdmf
# ...
hdf5_xdmf.Xdmf(...)
~~~

Reply:
I was aiming for shorter lines of code with the intention of easier readability. I think I will leave it like this.